inspire/base/static/less/format/brief-results.less

```diff
@@ -107,4 +107,28 @@ hr {
 
 .record-brief-details {
     font-size: 14px;
+}
+
+.fa-spinner {
```

Comment:
There's no need to recreate the animation, you can use the built-in `fa-spin` class by font awesome: https://fortawesome.github.io/Font-Awesome/examples/ , check section "Animated icons"